apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/TransactionSummary.tsx

```diff
@@ -53,10 +49,10 @@ export const TransactionSummary = () => {
     {
       label: strings.swapFeesTitle,
       value: `${getTotalFees(
-        selectedPool?.batcherFee.quantity,
-        asQuantity(poolFee),
+        selectedPool.batcherFee,
+        selectedPool.fee,
+        createOrder.amounts.sell,
```

Comment:
@jorbuedo  I thought `getTotalFees` will not calculate `pool fee` from percentage anymore

Reply:
@SorinC6  The idea is to remove `getTotalFee`, this file is not updated yet. Need to be sure about the fee functions first, and clarify with product how to show the fees.